pages/GSoC.md

```diff
@@ -11,7 +11,7 @@ permalink: /GSoC/
 
 Thank you for your interest in participating in the [Google Summer of Code program][gsoc] with the [Qubes OS team][team]. You can read more about the Google Summer of Code program at the [official website][gsoc] and the [official FAQ][gsoc-faq].
 
-Being accepted as a Google Summer of Code student is quite competitive. Students wishing to participate in the Summer of Code must be aware that you will be required to produce code for Qubes OS in 3 months. Your mentors, Qubes developers, will dedicate a portion of their time towards mentoring you. Therefore, we seek candidates who are committed to helping Qubes long-term and are willing to do quality work and be proactive in communicating with your mentor.
+Being accepted as a Google Summer of Code student is quite competitive. Students wishing to participate in the Summer of Code must be aware that you will be required to produce code for Qubes OS for 3 months. Your mentors, Qubes developers, will dedicate a portion of their time towards mentoring you. Therefore, we seek candidates who are committed to helping Qubes long-term and are willing to do quality work and be proactive in communicating with your mentor.
```

Comment:
Not to obsess over a single word, but I perceive a semantic difference.

"in 3 months" to me reads as "your code will exist in isolation, and at the end drop a patch", whereas "for 3 months" to me reads as "is working closely with the community throughout the duration". (The latter being favorable, especially in the context of the goals of GSoC.)

The GSoC org reviewers may interpret it the same way.